src/aiida_pseudo/cli/params/types.py

```diff
@@ -54,13 +55,34 @@ def complete(self, _, incomplete):
 
 
 class PseudoPotentialFamilyParam(GroupParamType):
-    """Parameter type for `click` commands to define an instance of a `PseudoPotentialFamily`."""
+    """Parameter type for ``click`` commands to define an instance of a ``PseudoPotentialFamily``."""
 
     name = 'pseudo_family'
 
+    def __init__(self, exclude: typing.Optional[typing.List[str]] = None, **kwargs):
+        """Construct the parameter.
+
+        :param exclude: an optional list of values that should be considered invalid and will raise ``BadParameter``.
```

Comment:
The word `exclude` seems a bit soft to me and doesn't immediately suggest that these will incur exceptions. Would `blacklist` maybe be better? As these are values that are actually blacklisted and not accepted?
```suggestion
        :param exclude: an optional list of values that should be considered invalid and will raise ``BadParameter``.
```

Reply:
Sure, `blacklist` also works. Or maybe `invalid`? Trying to use one of the "excluded" family types will indeed fail:

```console
$ aiida-pseudo family cutoffs set SSSP/1.1/PBE/efficiency metadata.json 
Usage: aiida-pseudo family cutoffs set [OPTIONS] FAMILY CUTOFFS
Try 'aiida-pseudo family cutoffs set -h' for help.

Error: Invalid value for 'FAMILY': The value `SsspFamily<SSSP/1.1/PBE/efficiency>` is not allowed for this parameter.
```

Reply:
I think `blacklist` is more correct since they aren't so much *invalid* as they are simply not allowed for this particular instance of the parameter type.